### PR TITLE
Attempt to reenable tests disabled because of messagepack issue.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Believed to be fixed by https://github.com/dotnet/roslyn/pull/66005.

### CODE REVIEW

**Comment by dibarbet** [comment]
/azp run

(need to trigger legs that don't normally run on PRs)

**Comment by dibarbet** [comment]
I forgot `/azp run` doesn't actually trigger the leg.  I've started a manual build here - https://dev.azure.com/dnceng-public/public/_build/results?buildId=112452&view=results - if the Debug_32 leg passes this PR should be good.

**Comment by dibarbet** [comment]
@CyrusNajmabadi this appears to be good to go!  No failures on the windows debug 32.  only hit known issue on coreclr single machine.

**Review by dibarbet — APPROVED** [review]

File: src/EditorFeatures/CSharpTest/AddUsing/AddUsingTests_ExtensionMethods.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.AddUsing
 {
     public partial class AddUsingTests
     {
-        [ConditionalFact(typeof(IsRelease), Reason = ConditionalSkipReason.TestIsTriggeringMessagePackIssue)]
+        [ConditionalFact]
```
Comment (jasonmalinowski) [inline src/EditorFeatures/CSharpTest/AddUsing/AddUsingTests_ExtensionMethods.cs:17]: Do these need to be switched back to regular facts?

**Review by jcouv — APPROVED** [review]
Compiler test-only change LGTM Thanks (iteration 6)

**Pull request merged**